.xo-config.json

```diff
@@ -48,6 +46,16 @@
         "svgo.config.mjs"
       ],
       "nodeVersion": ">=18"
+    },
+    {
+      "files": [
+        "scripts/add-icon-data.js",
+        "scripts/get-filename.js",
+        "scripts/release/update-sdk-ts-defs.js"
+      ],
+      "rules": {
+        "no-console": "off"
+      }
```

Comment:
I don't think so. It's the actual CLI part. There is no reason to enable `no-console` everywhere. I hope to get a comfortable development experience in the real CLI part instead of writing `eslint-disable` all the time.

![](https://github.com/simple-icons/simple-icons/assets/8186898/dabc59aa-7590-45e3-8dd3-798513e50090)

In the meantime, developers should use a modern debugging method such as `node—- inspect` rather than the `console.log`.

![](https://github.com/simple-icons/simple-icons/assets/8186898/daa2eeed-d25e-4198-b72d-d3425c868e91)

Reply:
Use `process.stdout.write` instead of `console.log`.